src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagEmailService.kt

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package nl.info.zac.productaanvraag
+
+import jakarta.enterprise.context.ApplicationScoped
+import jakarta.inject.Inject
+import net.atos.client.klant.KlantClientService
+import net.atos.client.klant.model.SoortDigitaalAdresEnum
+import net.atos.zac.admin.model.ZaakafhandelParameters
+import net.atos.zac.mailtemplates.MailTemplateService
+import net.atos.zac.mailtemplates.model.MailGegevens
+import nl.info.client.zgw.zrc.model.generated.Zaak
+import nl.info.zac.mail.MailService
+import nl.info.zac.mail.model.MailAdres
+import nl.info.zac.mail.model.getBronnenFromZaak
+import nl.info.zac.productaanvraag.exception.EmailAddressNotFoundException
+import nl.info.zac.productaanvraag.exception.InitiatorNotFoundException
+import nl.info.zac.productaanvraag.exception.MailTemplateNotFoundException
+import nl.info.zac.productaanvraag.model.generated.Betrokkene
+import nl.info.zac.util.AllOpen
+import nl.info.zac.util.NoArgConstructor
+import nl.info.zac.zaak.ZaakService
+
+@ApplicationScoped
+@NoArgConstructor
+@AllOpen
+class ProductaanvraagEmailService @Inject constructor(
+    private val klantClientService: KlantClientService,
+    private val zaakService: ZaakService,
+    private val mailService: MailService,
+    private val mailTemplateService: MailTemplateService,
+) {
+    fun sendEmailForZaakFromProductaanvraag(
+        zaakFromProductaanvraag: Zaak,
+        betrokkene: Betrokkene?,
+        zaakafhandelParameters: ZaakafhandelParameters
+    ) {
+        if (betrokkene == null) { throw InitiatorNotFoundException("No initiator provided") }
```

Comment:
Why this check here and why throw an exception? I would first do the check on `automaticEmailConfirmation` because if that is disabled, there is nothing to send anyway?

And we probably should not throw an exception here or should we? A product aanvraag without an initiator is normal behavior. It is not exceptional at all.